tests/bud.bats

```diff
@@ -551,6 +551,51 @@ _EOF
   expect_output "[container=buildah date=tomorrow]" "No Path should be defined"
 }
 
+@test "build with custom build output and output rootfs to directory" {
+  _prefetch alpine
+  mytmpdir=${TESTDIR}/my-dir
+  mkdir -p $mytmpdir
+  cat > $mytmpdir/Containerfile << _EOF
+FROM alpine
+RUN echo 'hello'> hello
+_EOF
+  run_buildah build --output type=local,dest=$mytmpdir/rootfs --signature-policy ${TESTSDIR}/policy.json -t test-bud -f $mytmpdir/Containerfile .
+  ls $mytmpdir/rootfs
+  # exported rootfs must contain `hello` file which we created inside the image
+  expect_output --substring 'hello'
+}
+
+@test "build with custom build output and output rootfs to tar" {
+  _prefetch alpine
+  mytmpdir=${TESTDIR}/my-dir
+  mkdir -p $mytmpdir
+  cat > $mytmpdir/Containerfile << _EOF
+FROM alpine
+RUN echo 'hello'> hello
+_EOF
+  run_buildah build --output type=tar,dest=$mytmpdir/rootfs.tar --signature-policy ${TESTSDIR}/policy.json -t test-bud -f $mytmpdir/Containerfile .
+  # explode tar
+  mkdir $mytmpdir/rootfs
+  tar -C $mytmpdir/rootfs -xvf $mytmpdir/rootfs.tar
```

Comment:
(If the goal is to only check file presence, using `tar -t` would allow listing the contents of the tarball without writing to disk. OTOH this actually verifies that the archive can be correctly extracted (assuming the tests fail on any failed command), which is valuable.)

Reply:
I didn't thought of any advantage here but I like the explanation on the second part hence keeping it as it is please feel free to unresolve this comment.